Command/CreateClientCommand.php

```diff
@@ -89,7 +106,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     private function buildClientFromInput(InputInterface $input): Client
     {
         $identifier = $input->getArgument('identifier') ?? hash('md5', random_bytes(16));
-        $secret = $input->getArgument('secret') ?? hash('sha512', random_bytes(32));
+
+        $isPublic = $input->getOption('public');
```

Comment:
@X-Coder264 Also, if this is not a bool flag, then shouldn't we cast to bool here?

Reply:
@HypeMC It's a bool flag.